src/components/compound/DlPagination/components/PageNavigation.vue

```diff
@@ -236,17 +248,16 @@ export default defineComponent({
             this.ellipsesEnd = false
 
             let maxDisplayRange = Math.max(
-                this.maxDisplayRange,
+                !this.maxDisplayRange || isNaN(Number(this.maxDisplayRange))
+                    ? 7
```

Comment:
Where did this number come from?

Reply:
It is an opinionated fallback. It shows current page, one neighbor page on either side and either boundary numbers or ellipsis on either side. And I also have made it into a component property (no magic values).